pages/add-user.page.ts

```diff
@@ -0,0 +1,86 @@
+import { Locator, Page } from "@playwright/test";
+import { extractSelectedDisplayedValue } from "../helpers/extractSelectedDisplayedValue";
+import { URLS } from "../config/urlProvider";
+
+export class AddUserPage {
+  readonly page: Page;
+  readonly createBtn: Locator;
+  readonly cancelBtn: Locator;
+  readonly userNameField: Locator;
+  readonly yearOfBirthField: Locator;
+  readonly genderField: Locator;
+  readonly headerListitem: Locator;
+  readonly maxUserNameLength: number;
+  readonly minUserNameLength: number;
+  public yearOfBirthFieldError: Locator;
+  public userNameFieldError: Locator;
+
+  constructor(page: Page) {
+    this.page = page;
+    this.createBtn = this.page.locator(
+      'xpath=//button[@data-testid="button-Create"]',
+    );
+    this.cancelBtn = this.page.locator(
+      'xpath=//a[@data-testid="button-Cancel"]',
+    );
+    this.userNameField = this.page.locator(
+      'xpath=//input[@id="inputUserName"]',
+    );
+    this.yearOfBirthField = this.page.locator(
+      'xpath=//input[@id="inputYearOfBirth"]',
+    );
+    this.genderField = this.page.locator('xpath=//select[@id="selectGender"]');
+    this.maxUserNameLength = 14;
+    this.minUserNameLength = 3;
+  }
+
+  async navigateToAddUserPage() {
+    await this.page.goto(URLS.ADDUSER);
+  }
+
+  async cancelAddUserOperation() {
```

Comment:
again, it will be better to create some generic method: clickButton and use this method for all needed buttons for the click. 
But anyway, about the click, as it's only one code line, you can not to create any methods for this purpose, you can just use it in a code: 

  await addUserPage.createBtn.click();